app/src/components/landing/Home/HomeV2/index.tsx

```diff
@@ -0,0 +1,127 @@
+        <div className="home-v2-feature-grid-item">
+          <GridHeader
+            title="Development"
+            description="Use the requestly tools to work with private servers and public APIs"
+            icon={developmentIcon}
+          />
+          <FeatureCard
+            title="Make an API request"
+            description="Make a request to an API by specifying endpoint and other request attributes or import from cURL"
+            tag={FEATURES.API_CLIENT}
+            navigateTo={PATHS.API_CLIENT.RELATIVE}
+            analyticsContext="api_client"
+          />
+          <FeatureCard
+            title="Create a mock API"
+            description="Generate custom API responses without actually having a pre-built API or backend server"
+            tag={FEATURES.MOCK_V2}
+            navigateTo={PATHS.MOCK_SERVER_V2.RELATIVE}
+            analyticsContext="mock server"
+          />
+        </div>
+        <div className="home-v2-feature-grid-item">
+          <GridHeader
+            title="Testing"
+            description="Edit incoming & outgoing request headers & bodies "
+            icon={testingIcon}
+          />
+          <FeatureCard
+            title="Modify network requests"
+            description="Create rules to modify HTTP requests & responses - URL redirects, Modify APIs or Headers"
+            tag={FEATURES.RULES}
+            navigateTo={PATHS.RULES.MY_RULES.ABSOLUTE}
+            analyticsContext="http_rules"
+          />
+        </div>
+        <div className="home-v2-feature-grid-item">
+          <GridHeader
+            title="Debugging"
+            description="Validate your solutions using screen recording, delaying responses & more"
+            icon={debuggingIcon}
+          />
+          <FeatureCard
+            title="Debug faster with Session Recording"
+            description="Capture screen, mouse movement, network, console and more of any browser session."
+            tag={FEATURES.SESSION_RECORDING}
+            navigateTo={PATHS.SESSIONS.RELATIVE}
+            analyticsContext="session_recording"
+          />
+        </div>
+      </div>
+    </div>
+  );
+};
+
+const GridHeader: React.FC<GridHeaderProps> = ({ title, description, icon }) => {
+  return (
+    <div className="home-v2-grid-header">
+      <img className="home-v2-grid-header-icon" src={icon} alt={title} />
+      <div>
+        <Typography.Title className="home-v2-grid-header-title">{title}</Typography.Title>
+        <Typography.Text className="home-v2-grid-header-subtitle">{description}</Typography.Text>
+      </div>
+    </div>
+  );
+};
+
+const FeatureCard: React.FC<FeatureCardProps> = ({ title, description, tag, navigateTo, analyticsContext }) => {
+  const navigate = useNavigate();
+  return (
+    <div
+      className="home-v2-grid-card"
+      onClick={() => {
+        trackEcosystemFeatureClicked(analyticsContext);
+        navigate(navigateTo);
```

Comment:
Let's change this to a `<Link />` else user won't be able to open the feature page in the new tab using `CMD + click`